strax/storage/s3.py

```diff
@@ -63,10 +67,21 @@ def __init__(self,
             aws_secret_access_key = os.environ.get('AWS_SECRET_ACCESS_KEY')
 
         #  Initialized connection to S3-protocol storage
-        self.s3client = boto3.client(aws_access_key_id=aws_access_key_id,
-                                     aws_secret_access_key=aws_secret_access_key,
-                                     endpoint_url=endpoint_url,
-                                     service_name='s3')
+        self.s3 = boto3.client(aws_access_key_id=aws_access_key_id,
+                               aws_secret_access_key=aws_secret_access_key,
+                               endpoint_url=endpoint_url,
+                               service_name='s3')
+
+        # List of all buckets on this S3 store (finally, a bucket list!)
+        # then determine if either no buckets exist or (if they exist)
+        # whether or not the bucket we need exists.  If not, create.
+        # bucket_list = self.s3.list_buckets()
```

Comment:
Is this commented out code still needed?

Reply:
Fixed